# Parámetros para el Nodo que calcula el Error
dist_finder:
  ros__parameters:
    #desired_distance: 0.8       # Metros a la pared
    desired_distance: 0.94
    desired_wall_side: -1        # 1: Izquierda, -1: Derecha
    lookahead_dist: 1.55      # Metros a proyectar a futuro (
    theta: 1.075            # Ángulo de apertura (60 grados en radianes)
    # Nota: ray_b_angle se calcula en el nodo basado en el lado, 
    # pero si quisieras forzarlo, podrías agregarlo aquí.

# Parámetros para el Nodo de Control (PID)
control:
  ros__parameters:
    kp: 1.8                     # Ganancia Proporcional (Fuerza de giro)
    kd: 0.6                     # Ganancia Derivativa (Amortiguación)
    max_speed: 3.3              # Velocidad máxima en rectas (m/s)
    max_steering: 4.0           # Límite físico de las llantas (rad)